### PR TITLE
Add dynamic scheduled tasks doc

### PR DESCRIPTION
**Pull request by ninayang**
@jpwain let me know if the doc is clear? I tried to show more through code than words.

I imagine this to be a page right below the current Scheduled Task page, in the docs sidebar.

### CODE REVIEW

File: building/dynamic-scheduled-tasks.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+id: dynamic-scheduled-tasks
+title: Dynamic Scheduled Tasks
```
Comment (jpwain) [inline building/dynamic-scheduled-tasks.md]:
```suggestion
title: Dynamic scheduled tasks
```
No need to capitalize here.

File: building/dynamic-scheduled-tasks.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+layout: docs.mustache
+tags: doc
+---
+
+You can dynamically schedule an operation to run using a Dynamic Task. These tasks are managed exclusively through JavaScript operations.
```
Comment (jpwain) [inline building/dynamic-scheduled-tasks.md]:
```suggestion
You can dynamically schedule an operation to run using a dynamic scheduled task. These tasks are managed exclusively through JavaScript operations.
```
Normalize capitalization and phrasing.

File: building/dynamic-scheduled-tasks.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  return task.create("this.get_calendar_events").runEvery(1, "hour");
+}
+
+// Operation get_calendar_events
+(params) => {
+  return api.run("google_calendar.get_calendar_events", {calendarId: "primary"});
+}
+```
+
+### Creating a task for all your users
```
Comment (jpwain) [inline building/dynamic-scheduled-tasks.md]:
```suggestion
### Creating a task for all registered application users
```

File: building/dynamic-scheduled-tasks.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+The following operations will create a task to return calendar events for all of your users every hour:
```
Comment (jpwain) [inline building/dynamic-scheduled-tasks.md]:
```suggestion
The following operations will create a task to return calendar events for all application users every hour:
```

File: building/dynamic-scheduled-tasks.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  return task.create("this.get_calendar_events_for_everyone").runEvery(1, "hour");
+}
+
+// Operation get_calendar_events_for_everyone:
+(params) => {
+  return api.runForAllUsers("google_calendar.get_calendar_events", {calendarId: "primary"})
+}
+```
+
+## View your created dynamic tasks
```
Comment (jpwain) [inline building/dynamic-scheduled-tasks.md]:
```suggestion
## List existing dynamic tasks
```

File: building/dynamic-scheduled-tasks.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+`task.listTasks()` will return `runEvery` dynamic tasks, as well as `runOnce` tasks that have not yet executed.
+
+## Delete a dynamic task
+```javascript
+var uuid = task.create("this.helloworld").runEvery(5, "minutes");
+task.delete(uuid);
+```
+
+You can delete all tasks (in your current context) by running the following:
```
Comment (jpwain) [inline building/dynamic-scheduled-tasks.md]:
```suggestion
You can delete all dynamic tasks (in your current context) by running the following:
```
Unclear to me if this means all tasks or just dynamic tasks; disregard if this suggestion doesn't apply.

**Comment by ninayang** [comment]
@jpwain I'd prefer to keep it two separate documents, I think it's easier on the developer's mind that way, especially since you can't access one from the other

**Comment by jpwain** [comment]
SGTM re: separate page

**Comment by ninayang** [comment]
@jpwain Are you good with this doc? (I made all the changes you suggested). I'm going to have Tuan merge it in when he's done with his credentials refactor

**Comment by jpwain** [comment]
@ninayang Yep, @tuan-nguyen-tps merge when appropriate.

**Pull request merged**